tests/end_to_end/candid_rpc/class_syntax/motoko_examples/cert-var/src/index.ts

```diff
@@ -1,11 +1,14 @@
-import {
-    candidEncode,
-    dataCertificate,
-    IDL,
-    query,
-    setCertifiedData,
-    update
-} from 'azle';
+import { dataCertificate, IDL, query, setCertifiedData, update } from 'azle';
+
+type CertifiedValue = {
+    value: number;
```

Comment:
TypeScript type for value and IDL do not match

Reply:
done